bower.json

```diff
@@ -21,6 +21,8 @@
     "firebase-element": "PolymerElements/firebase-element#^1.0.12"
   },
   "resolutions": {
+    "iron-location": "1 - 2",
```

Comment:
D'oh. Had no idea we could do it so simply. In the docs I think we currently have something like `^1.9.1 || ^2.0.0` ... And we could get away with `1 - 2`? sheesh.